release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -83,7 +83,12 @@ For more information on required permissions, see xref:../installing/installing_
 ====
 ==== Installing a cluster on {aws-short} by using an existing IAM profile
 
-With this release, you can install {product-title} on {aws-first} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional {aws-short} configuration parameters].
+With this release, you can install {product-title} on {aws-first} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional {aws-short} configuration parameters].
+
+[id="ocp-4-17-installation-and-update-gcp-n4_{context}"]
+==== Installing a cluster on {gcp-short} using the N4 machine series
+
+With this release, you can deploy a cluster on {gcp-short} using the link:https://cloud.google.com/compute/docs/general-purpose-machines#n4_series[N4 machine series] for compute or control plane machines. Machines that use the N4 instance type use the `hyper-balanced` disk type. For more information, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-gcp[Installation configuration parameters for GCP].
```

Comment:
The supported disk type of **N4** machine series is `hyperdisk-balanced` instead, thanks!